eslint.config.js

```diff
@@ -13,6 +14,17 @@ module.exports = [
 		},
 	},
 	n.configs["flat/recommended"],
+	{
+		rules: {
+			// TS handles this for us
+			"n/no-missing-import": "off",
+			// Using a ts bin file throws this rule off.
+			// it uses the package.json as a source of truth, and since the package points
+			// at the transpiled js file, it treats usage on the ts src as a violation.
+			"n/hashbang": "off",
```

Comment:
Heh, this is why I normally still include a `bin/*` file that imports a `lib/*` file: https://github.com/JoshuaKGoldberg/prune-github-notifications/blob/247c5cc477b063d6772823ff6f37c2207b1095c7/bin/index.js

I personally prefer having a small `bin` file the way that repo does. But I don't feel strongly. I suppose either we have a more complicated ESLint config or an extra folder + file. If you strongly prefer the current approach, I don't mind. 🙂

Reply:
One of the downsides to that pattern, and why I don't generally go with it, is that the bin/*.js file is importing from a file that will only exist after build, which impacts our ability to do some amount of static analysis on it.

Reply:
👍 no complaint from me for now. Thanks!